server/services/interventionsService.test.ts

```diff
@@ -1796,6 +1802,66 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
         activities,
         submittedAt: '2021-03-09T15:08:38Z',
         submittedBy: user,
+        approvedAt: null,
+        approvedBy: null,
+      })
+    })
+
+    it('returns an existing approved action plan', async () => {
+      const actionPlanId = 'f3ade2c5-075a-4235-9826-eed289e4d17a'
+      const referralId = '1BE9F8E5-535F-4836-9B00-4D64C96784FD'
+      const user = {
+        authSource: 'auth',
+        userId: 'BB9C99F7-13EA-43D0-960A-768DC8FA0D91',
+        username: 'SP_USER_1',
+      }
+      const activities = [
+        {
+          id: '91e7ceab-74fd-45d8-97c8-ec58844618dd',
+          description: 'Attend training course',
+          desiredOutcome: {
+            id: '301ead30-30a4-4c7c-8296-2768abfb59b5',
+            description:
```

Comment:
This will be removed in future (done in my PR) where we're not linking activities to desired outcomes, but we can probably merge this first as the API change isn't in yet.